src/config.rs

```diff
@@ -184,6 +206,28 @@ fn default_background() -> Rgba<f32> { Rgba {r: 0.0, g: 0.0, b: 0.0, a: 0.0} }
 mod tests {
     use super::*;
 
+    #[test]
+    fn resolve_absolute_path() {
+        let base_path = "/spritec/sample/bigboi";
+        let input_path = "/spritec/sample/bigboi/test";
+        let output_path = "/spritec/sample/bigboi/test";
+        resolve_check(base_path, input_path, output_path);
+    }
+
+    #[test]
+    fn resolve_relative_path() {
+        let base_path = "/spritec/sample/bigboi";
+        let input_path = "../../src/bin";
```

Comment:
I would like to see a test where there are more ".." than there are parent directories and also one where you mix both ".." and "." syntax (normal usage and some weird combinations you wouldn't normally see)